modules/retail/pages/containerized-saltboot.adoc

```diff
@@ -0,0 +1,158 @@
+
+Name of the [systemitem]``Saltboot group`` is automatically used as a [systemitem]``branch id``, an identifier for group of machines booted through particular containerized {productname} Proxy.
+
+All [systemitem]``saltboot`` deployed machines though containerized proxy will automatically became members of its [systemitem]``Saltboot group``.
+
+To connect [systemitem]``Saltboot group`` with containerized proxy fill [systemitem]``Image Download Server`` entry with Fully Qualified Domain Name ([literal]``FQDN``) of the containerized proxy.
+
+The rest of configuration is optional.
+
+=== Default boot image
+
+Configure [systemitem]``Default boot image for new registrations`` to specify what boot image should be booted by not yet registered POS terminal.
+This is useful when stable boot image is wanted for initial deployments.
+Without this setting, newest built boot image is used as default boot image.
+
+If [systemitem]``Default boot image for new registrations`` is set, option to set its version appears under name [systemitem]``Default boot image version`` where specific image version can be set.
+
+=== Kernel option for the saltboot group
+
+Option [systemitem]``Kernel parameters for the group`` can be used to pass extra kernel options to all POS terminals registered withing this saltboot group.
+
+=== Naming scheme for new registrations
+
+Last three options are related to how will be newly registered machine visible in {productname} Server.
+
+See xref:retail-terminal-names.adoc[] for explanation of possible configurations.
+
+== Comparing containerized and non-containerized workflows
+
+External DHCP service must be used with containerized Saltboot.
+
+Containerized workflow relies on updated image building in {productname} Server 4.3 where PXE images are no longer collected as bundle, but kernel, initrd and filesystem image are collected individually.
+
+Containerized workflow uses new TFTP container which instead of providing files present on the proxy, routes specific TFTP requests as HTTP requests through local squid proxy to {productname} Server.
+
+Containerized proxy is not a Salt client, it is not possible to call [systemitem]``image-sync`` state.
+
+[IMPORTANT]
+====
+Once POS image is build and made available on {productname} Server, it is immediately available to the Saltboot clients as well.
+This may have implications on how images are deployed to production.
+====
+
+The following sections differentiate between containerized and regular workflow.
+Both are assuming proxy (containerized or in form of {productname} {smr} Branch Server) are available.
+
+Containerized workflow:
+
+. Build POS image
+. Configure DHCP server for PXE booting for given network
+. Create [systemitem]``Saltboot group`` and configure it for existing containerized proxy
```

Comment:
This section miss the external DHCP, and the need of a system for saltboot formula.